src/test/java/seedu/heymatez/commons/util/StringUtilTest.java

```diff
@@ -16,33 +17,33 @@
     public void isNonZeroUnsignedInteger() {
```

Comment:
Similar to above, should this method name be modified such that it does not sound like a boolean?

Reply:
noted!